snakedoc/smk.py

```diff
@@ -109,7 +109,10 @@ def _gen_docs(viewlist: ViewList, rules: Mapping[str, snakemake.rules.Rule]):
         for rule in rules.values():
             lines = []
             lineno = rule.workflow.linemaps[rule.snakefile][rule.lineno]
-            lines.extend([f".. smk:rule:: {rule.name}", f"   :source: {Path(rule.snakefile).resolve()}:{lineno}", ""])
+            rule_type = "rule" if not rule.is_checkpoint else "checkpoint"
```

Comment:
It seems that checkpoints are just rule objects. (By the way - I am just writing this comment to test a neovim plugin!)